### DISCUSSION TITLE
10월 4일 중간 회고

### DISCUSSION THREAD

**Discussion by devHudi** [woowacourse-study/2022-cs-plant#31] [General]
1. 스터디 방향이 목적대로 잘 가고 있는지
2. 본인이 스터디에 열심히 참여하고 있는지?
3. 스터디 방식에서 개선하고 싶은 점이 있는지?
4. 스터디가 본인 성장에 도움이 되는지?

**Comment by gudonghee2000** [comment]
- 스터디 방향이 목적대로 잘 가고 있는지
목적과 같이 앞으로의 진로를 위해 방향성이 잘 나아가고 있는것 같다.

- 본인이 스터디에 열심히 참여하고 있는지?
너무나도 열심히 참여하고 싶다.

- 스터디 방식에서 개선하고 싶은 점이 있는지?
개선하고 싶은 부분은 아직 없다.

- 스터디가 본인 성장에 도움이 되는지?
정말 많이 도움이돼고 지식이 성장하고 있음을 느낀다.

**Reply by devHudi** [reply]
맞아요. 면접관 입장에서 질문하는 입장이 되어보니 자연스레 꼬리질문을 생각하게 되는 것 같아요.

**Comment by hyeonic** [comment]
1. 스터디 방향이 목적대로 잘 가고 있는지
    * 인터뷰 형식에 맞게 잘 진행되고 있다.
2. 본인이 스터디에 열심히 참여하고 있는지?
    * 솔직히 아니다. 매번 급하게 내용은 채우는 듯한 느낌을 지울 수 없다.
3. 스터디 방식에서 개선하고 싶은 점이 있는지?
    * 현재 방식을 유지하는 게 가장 좋아 보인다. 하지만 좀 더 마음가짐이 필요할 것 같다. 
4. 스터디가 본인 성장에 도움이 되는지?
    * 도움이 많이 된다. 다만 이렇게만 준비해도 괜찮을지는 조금 무섭다.

**Comment by devHudi** [comment]
1. 스터디 방향이 목적대로 잘 가고 있는지

스터디의 목적은 프로젝트, 미션, 실습과 관련된 CS 지식을 모의 면접의 형태로 진행하며, 그 역량을 키우기 위함이라고 생각합니다.
방향 자체는 잘 흘러가고 있는 것 같으나, 조금 더 CS 지식과 미션 내용을 관련지어 진행되면 더 좋겠습니다.

2. 본인이 스터디에 열심히 참여하고 있는지?

나쁘지 않지만, 시간이 다소 촉박하게 제출하고 있는 것 같습니다.

3. 스터디 방식에서 개선하고 싶은 점이 있는지?

조금 더 미션과 프로젝트에 관련지어 내용을 작성해오고, 질문도 단순 개념 물어보기가 아니라 직접 경험한 내용을 이야기할 수 있도록 흘러가면 좋을 것 같습니다.

4. 스터디가 본인 성장에 도움이 되는지?

다른 것 보다 말하기 능력과 면접 상황에 익숙해지는게 도움이 되는 것 같습니다.

**Reply by devHudi** [reply]
- 금요일에 마감하는거 좀 힘들다.

**Reply by devHudi** [reply]
3번에 추가하자면, 늦거나 안내거하는거에 대한 패널티가 없어서 패널티를 확실히 줘야할 것 같다.

**Comment by devHudi** [comment]
1. 스터디가 좀 벅찬것 같은 느낌이 든다.
    - 주에 각자 개념 1개씩으로 하향 조정하기
    - 주에 3명이 전부 최대 20분정도 실제와 같은 모의면접을 진행하기
    - 레벨4 마지막즈음 지금까지 이슈에 쌓아온 지식들로 1시간 정도 긴 모의 면접을 진행한다.
2. 스터디가 끝나면 회고와 학습 내용 정리, 토론 꼭 하기 (10분정도라도)
3. 질문 방향
    - 왜 도입, 왜 썼는지
    - 프로젝트에 어떻게 적용해보고 싶은지

    등등 프로젝트와 경험과 많이 연관지어서 질문하기
4. 학습 방향
    - 기술을 왜 도입했는지? -> 그 기술이 없었으면 우린 어떤 불편함을 겪었을지? -> 왜 그것이 표준일지? -> 왜 스프링은 그 기술을 기본으로 선택했는지

    이렇게 깊게 학습하는 태도가 필요함
5. 레벨 5 계획
    - 엄청 자주 모의 면접 진행하기
    - 상세한 진행 계획은 레벨4 마지막에 결정
